radio/src/targets/horus/telemetry_driver.cpp

```diff
@@ -28,6 +28,11 @@ DMAFifo<TELEMETRY_FIFO_SIZE> telemetryDMAFifo __DMA (TELEMETRY_DMA_Stream_RX);
 uint8_t telemetryFifoMode;
 #endif
 
+//OW
+extern Fifo<uint8_t, 32> mavlinkTelemExternalTxFifo_frame;
```

Comment:
Is there a possibility the same buffers could be used as other telemetry protocols, or is that necessary on top of the others?

Reply:
in principle it is possible to get away with less buffers
but that's btw so also for the telemetry protocols themselves, it's quite a mess frankly
what it actually would need is a pretty complete overhaul of the whole protocol and and module and serial handling. IMHO.

Reply:
I don't expect you to use smaller buffers! I just thought we could have the buffer size being some "#define", and have their size being cranked up when compiled with MAVlink support. Does that make sense?

Reply:
I'm somewhat lost here too ... my answer would be just a rewording of my previous answer, not sure what else I should say

this can be done in 10000 ways, and 9000 of them are equally good ...

I'm not a friend of what you suggest because this just leads to exactly this miriad of tiny flags doing tiny things ... I rather would go with what I suggested, but this shouldn't be part of this PR
it's frankly also somewhat ironical since you here suggest to add flags for doing tiny things yet constantly complain about each flag that they should by run time ... it's still a mistery to me what architecture you actually strive for ... if there is such a plan